assets/javascripts/discourse/templates/components/payment-plan.hbs

```diff
@@ -1,6 +1,8 @@
 <DButton
-  @action={{action "planClick"}} @class={{concat "btn-discourse-subscriptions-subscribe " selectedClass}}>
-  <div class="interval">
+  @action={{action "planClick"}}
+  @class={{concat "btn-discourse-subscriptions-subscribe " selectedClass}}
```

Comment:
```suggestion
  @class={{concat "btn-discourse-subscriptions-subscribe" selectedClass}}
```

Reply:
This does not work because the two classes will be concatenated and result in a very long string `btn-discourse-subscriptions-subscribebtn-primary`.

Reply:
Ah, I was certain you were using `concat-class` here already! 😃 What I had in mind was actually:

```suggestion
  @class={{concat-class "btn-discourse-subscriptions-subscribe" selectedClass}}
```

Reply:
Oh, nice! I did not know about that helper. TIL